src/main/java/com/vonage/client/messages/whatsapp/Context.java

```diff
@@ -57,7 +57,10 @@ public UUID getMessageUuid() {
 	 * Only applies to Order messages.
 	 *
 	 * @return The referred product details, or {@code null} if not applicable.
+	 *
+	 * @deprecated This will be moved in a future release.
```

Comment:
Do you mean 'removed'?

```suggestion
	 * @deprecated This will be removed in a future release.
```

Reply:
No I mean moved, because it's now part of the `whatsapp` object in the spec.